geode-dunit/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheServerTestUtil.java

```diff
@@ -508,6 +494,11 @@ public boolean waitWhileNotEnoughEvents(long sleepMs, int eventCount, int eventT
       return waitWhileNotEnoughEvents(sleepMs, eventCount, getEvents(eventType));
     }
 
+
+    /**
+     * This method is not implemented to test event count matches the eventsToCheck.size() which is
+     * confusing. It will wait and return if it got something in the eventsToCheck or not.
+     */
```

Comment:
This should either use correct javadoc semantics (add a `@param` tag for each argument and an `@return` tag for the returned value) or it should be a block comment rather than a javadoc comment.

Reply:
I don't agree, I would rather have the hover over documentation with some information than none. Though completing the information should not be that hard.

Reply:
I just have a bee in my bonnet about javadocs after the big clean-up I did lately, I think. I saw so many places where there was stuff like
```
/**
* Constructs the object
* /
public SomeClass(int param1, int param2) {
...
```
that provided no value as a javadoc and caused the linter to throw warnings. I think sometimes people just use javadoc comments because they're green in IntelliJ and so stand out more.